scenes/enumerated/problem_11273/problem_11273.py

```diff
@@ -0,0 +1,246 @@
+from manim import *
+from lib.qarakusiscene import TaskNumberBox
+from .text import *
+
+class Problem11273(Scene):
+    def construct(self):
+        scale = .6
+        task_number = TaskNumberBox(task_number_string)
+        rectangle1 = Rectangle(height=3, width=9).scale(scale)
+        rectangle2 = Rectangle(height=3, width=3).scale(scale).align_to(rectangle1, LEFT)
+        rectangle3 = Rectangle(height=3, width=6).scale(scale).align_to(rectangle1, RIGHT)
+        example1 = Rectangle(height=2, width=10).scale(scale)
+        example2 = Rectangle(height=1, width=11).scale(scale)
+        example1_height_label = MathTex(r"2\text{"+cm+"}").next_to(example1, LEFT)
+        example1_width_label = MathTex(r"10\text{"+cm+"}").next_to(example1, UP)
+        example2_height_label = MathTex(r"1\text{"+cm+"}").next_to(example2, LEFT)
+        example2_width_label = MathTex(r"11\text{"+cm+"}").next_to(example2, UP)
```

Comment:
```suggestion
        example1_height_label = MathTex(rf"2\text{cm}").next_to(example1, LEFT)
        example1_width_label = MathTex(rf"10\text{cm}").next_to(example1, UP)
        example2_height_label = MathTex(rf"1\text{cm}").next_to(example2, LEFT)
        example2_width_label = MathTex(rf"11\text{cm}").next_to(example2, UP)
```
I'm not 100% sure, but you can check if you like this